assets/css/styles.css

```diff
@@ -42,9 +42,8 @@ a {
 
 #error-message {
   color: red;
-  font-size: 1.5rem;
+  font-size: 25px;
   margin-bottom: 1.5rem;
-  
 }
```

Comment:
- [ ] Your email validation works fine :+1:  but from the project requirement, If the validation is not OK, you should show an error message to the user **near the submit button** informing them of the error, and the form is not sent. So make sure you display the error message near the submit button. As you can see below your error message is not near the submit button :smile: 

  ![image](https://user-images.githubusercontent.com/92303891/233807376-666244bf-4c9a-48de-98d6-ca149bd6f2ea.png)

Reply:
Thanks, dear friend **@Owusu-Desmond** I'll fix it immediately. 

Javier.-